### PR TITLE
Adding TikhonovSVD and TSVD example networks

### PR DESCRIPTION
**Pull request by jcollfont**
Added new example networks for TikhonovSVD and TSVD for completeness

### CODE REVIEW

**Comment by jessdtate** [comment]
I'll merge this after we merge the tikhonov code into scirun.

**Comment by jessdtate** [comment]
can you change the test_toolkit_file.py to reflect the new number of networks (line 68)? It's a hard coded check that we need to improve.  You will also need to rebuild the toolkit file using the bundle_toolkit code in the SCIRun/bin/SCIRun directory.

**Comment by jcollfont** [comment]
Updated the number of networks in test_toolkit_file.py. 

The bundle_toolkit includes the toolkit in the SCIRun UI, right? I hadn’t done this before.

You are right, all the new networks (including isotropy method) should be applied after the merge in SCIRun with the new Tikhonov modules

**Comment by jessdtate** [comment]
the bundle_toolkit script will make the toolkit to include in the SCIRun UI, yes.  It's an extra step, but it is at least scripted.  Just run 
`{path to scirun bin scirun}/bundle_toolkit Networks/FwdInvToolkit.toolkit  Networks`

**Comment by jcollfont** [comment]
got it! It worked :)

File: test_toolkit_file.py
Comment (dcwhite) [inline test_toolkit_file.py]: This is not where you update number of networks. It's below...

File: test_toolkit_file.py
Comment (dcwhite) [inline test_toolkit_file.py]: ...here. Please pay attention to the comments.

**Review by dcwhite — CHANGES_REQUESTED** [review]

File: Documentation/ECGTool_inverse.tex
Comment (jessdtate) [inline Documentation/ECGTool_inverse.tex:131]: was this file added to the repo?

File: Documentation/ECGTool_inverse.tex
Comment (jessdtate) [inline Documentation/ECGTool_inverse.tex:176]: This seems to be missing too

File: Documentation/ECGTool_inverse.tex
Comment (jessdtate) [inline Documentation/ECGTool_inverse.tex:200]: and this one

File: Documentation/ECGTool_inverse.tex
Comment (jessdtate) [inline Documentation/ECGTool_inverse.tex:246]: and this

File: Documentation/ECGTool_inverse.tex
Comment (jessdtate) [inline Documentation/ECGTool_inverse.tex:288]: this too

File: Documentation/ECGTool_inverse.tex
Comment (jessdtate) [inline Documentation/ECGTool_inverse.tex:480]: this one

**Comment by jessdtate** [comment]
The images referenced are missing from the repo

**Pull request merged**
